### ISSUE TITLE
Make testing clearer

### ISSUE THREAD

**Issue by dmuth** [Comcast/snowdrift#8]
The timeout messages in the testing script look scary because they're in red, when the truth is, they're actually expected and part of the testing process:

![image](https://user-images.githubusercontent.com/374060/107859500-e0c89400-6e07-11eb-803e-cc00e39ac89f.png)

I want to add in assertion checking on each specific test, and if the assertion passes, the text is printed in green.  If it fails, the text is printed in red, with an explanation.

**Comment by steinbrueckri** [comment]
@dmuth Just FYI, i work on BATs for the testing of snowdrift, so before you start doing anything give me 2-3 weeks more and I will come back to you with a PR.

**Comment by dmuth** [comment]
I'm not sure what BAT means, but I'll commit work to my branch, submit it as a PR, and we can go over to together and decide how to proceed.

**Comment by steinbrueckri** [comment]
> I'm not sure what BAT means, but I'll commit work to my branch, submit it as a PR, and we can go over to together and decide how to proceed.
> 
> 

Ref. for BATs: https://github.com/bats-core/bats-core

**Comment by dmuth** [comment]
Interesting!

I was able to finish up my PR, which can be viewed over at #10.  The big reason why I wrote it was so that tests looked a little cleaner, and we didn't have issues with updating assertions, which really caused us some challenges towards the end of #7 and made the submitter have to do more work than they should have.

Unless you think the changes in #10 will totally derail what you have in mind, I'd like to integrate 'em.
